pt-BR/mail/chrome/messenger/am-server-advanced.dtd

```diff
@@ -22,7 +22,7 @@
 <!ENTITY overrideNamespaces.label "Permitir que o servidor sobrescreva estes namespaces">
 <!ENTITY overrideNamespaces.accesskey "e">
 <!ENTITY pop3Desc.label "Ao receber mensagens POP deste servidor, salvá-las na:" >
-<!ENTITY folderStorage.label "Where to store mail">
+<!ENTITY folderStorage.label "Onde armazenar emails">
```

Comment:
@brunotiussi I understand your position, however my recommendation is to translate all possible words to PT-BR. If you'd like to keep it short (GUI purposes) either use e-mail or email (chose one to keep consistency).